Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM kernai/refinery-parent-images:v1.11.0-common
 
 WORKDIR /app
 
+RUN apt-get update && apt-get install -y libc6-dev zlib1g gcc --no-install-recommends
```

Comment:
Is there a specific reason this is done via the Dockerfile and not the requirements.txt? (They are linux packages vs python packages, I ask why we don't use python packages)

Also libc6-dev sounds like a dev version of a package. Is this the correct one?

Reply:
yes will do it!